docs/user/faq.rst

```diff
@@ -96,12 +98,6 @@ Therefore, as long as you implement these classes and callables in a
 thread-safe manner, and ensure that any third-party libraries used by your
 app are also thread-safe, your WSGI app as a whole will be thread-safe.
 
-That being said, IO-bound Falcon APIs are usually scaled via multiple
```

Comment:
I decided to remove this since we really haven't seen any "edge cases" over the years, unless I'm missing something. If so, we can state explicitly what those are.